pkg/auth/auth_test.go

```diff
@@ -31,7 +31,7 @@ var largeAuthFileValues = map[string]types.DockerAuthConfig{
 
 // systemContextForAuthFile returns a types.SystemContext with AuthFilePath pointing
 // to a temporary file with fileContents, or nil if fileContents is empty; and a cleanup
-// function the calle rmust arrange to call.
+// function the called rmust arrange to call.
```

Comment:
Should be caller, not called. And there's an extra r on the must

Reply:
Strange that codespell missed that.